.github/buildomat/jobs/tuf-artifact.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#:
+#: name = "helios / tuf-artifact"
+#: variety = "basic"
+#: target = "helios-latest"
+#: rust_toolchain = "nightly-2022-09-27"
+#: output_rules = [
+#:	"=/work/repo.zip",
+#: ]
+#:
+#: [dependencies.package]
+#: job = "helios / package"
+#:
+
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+cargo --version
+rustc --version
+
+COMMIT=$(git rev-parse HEAD)
+VERSION="0.0.0+git${COMMIT:0:11}"
+
+pushd /input/package/work/out
+tar cvf "/work/control-plane.tar" ./*
+popd
+
+cargo build --locked --release --bin tufaceous
+
+python3 -c 'import secrets; open("/work/key.txt", "w").write("ed25519:%s\n" % secrets.token_hex(32))'
+read -r TUFACEOUS_KEY </work/key.txt
+export TUFACEOUS_KEY
+
+target/release/tufaceous -r /work/repo init --no-generate-key
+target/release/tufaceous -r /work/repo add --name control_plane control_plane "/work/control-plane.tar" "$VERSION"
+target/release/tufaceous -r /work/repo archive /work/repo.zip
```

Comment:
So I can see how we're labeling *components* with versions - how do you think this should map to a system-level version?

Reply:
(Not trying to place the pressure of "making the call about what the system version should be" here - if we use a schema like `0.0.0+git commit`, that totally works too, just wondering how we shove that all into a TUF repo that Nexus could theoretically query later)

Reply:
I think actually this shouldn't map to a system-level version at all, at this stage. That seems like a release engineering process, to decide which artifacts (from all the repositories) become a specific ✨ System Version ✨.

(And that process can be automatic, and can be based on the commit hashes of the various components, so that we can have a nightly system version or the like. But the point is that the ✨ System Version ✨ is a combination of things that's outside the control of (probably) this repo.)